transcribe.py

```diff
@@ -41,12 +47,23 @@ def main(args):
 	decoder = decoders.GreedyDecoder() if args.decoder == 'GreedyDecoder' else decoders.BeamSearchDecoder(labels, lm_path = args.lm, beam_width = args.beam_width, beam_alpha = args.beam_alpha, beam_beta = args.beam_beta, num_workers = args.num_workers, topk = args.decoder_topk)
 	
 	data_paths = [p for f in args.input_path for p in ([os.path.join(f, g) for g in os.listdir(f)] if os.path.isdir(f) else [f]) if os.path.isfile(p) and any(map(p.endswith, args.ext))] + [p for p in args.input_path if any(map(p.endswith, ['.json', '.json.gz']))]
+	
+	if args.skip_processed:
```

Comment:
This can be simplified as:

```python
exlcude = set(basename for basename in os.listdir(args.output_path) if basename.endswith('.json')] if args.skip_processed else []
data_paths = [path for path in data_paths if os.path.basename(path) not in exclude]
```
I.e. no `find_preprocessed_paths()` function and no `if` block is needed